packages/repository-json-schema/src/__tests__/integration/build-schema.integration.ts

```diff
@@ -765,11 +766,12 @@ describe('build-schema', () => {
         @hasMany(() => Product)
         products?: Product[];
       }
-
+      const options: JsonSchemaOptions<Category> = {includeRelations: true};
```

Comment:
I feel the introduction of `options` property in the tests is adding too much complexity because we need to provide explicit type. I guess I can live with that though 🤷‍♂

Reply:
Simplified without options